### PR TITLE
Feature/fix dimensions

### PR DESCRIPTION
**Pull request by mvrsdev**
Added the dimensions information to the object.

### CODE REVIEW

File: scripts/car.js [hunk 1]
```diff
@@ -12,11 +12,17 @@ const userCar = {
...
-  // safety: 1234,
-  // performance: 1234,
-};
\ No newline at end of file
+  currentColor: 'Grey',
+  engine:{},
+  transmission:{},
+  imagePath:{},
+  dimensions:{
+    weight: 	1375, // In Kilograms
```
Comment (thi-days) [inline scripts/car.js:20]: WTHell is those spaces, bro?

File: scripts/car.js [hunk 2]
```diff
@@ -12,11 +12,17 @@ const userCar = {
...
+    height: 1484, // In millimeters
+    width: 1823, // In millimeters
+    length: 4358, // In millimeters
+    betweenAxles: 	2648 // In millimeters
```
Comment (thi-days) [inline scripts/car.js:24]: The same bluuuuur here. Lol. 😂

**Review by thi-days — COMMENTED** [review]
Change base branch to develop.

**Comment by mvrsdev** [comment]
Now is really fixed.

**Review by thi-days — APPROVED** [review]

**Pull request merged**